src/Thumbnails/Thumbnails.js

```diff
@@ -21,72 +20,70 @@ const useStyles = makeStyles((theme) => ({
     transform: 'translateZ(0)',
   },
   title: {
-    color: theme.palette.primary,
+    color: theme.palette.secondary,
+    fontSize: 24,
   },
   titleBar: {
     background:
       'linear-gradient(to top, rgba(0,0,0,0.7) 0%, rgba(0,0,0,0.3) 70%, rgba(0,0,0,0) 100%)',
   },
 }));
-
-// const residents = [
- 
-//   {
-//     img: 'https://previews.123rf.com/images/natakot/natakot1810/natakot181001281/110094284-happy-old-man-showing-thumbs-up-working-out-with-a-dumbbell.jpg',
-//     title: 'Ralph',
-//     author: 'author',
-//    },
-//    {
-//     img: 'https://erinwritesnow.files.wordpress.com/2017/01/cute-happy-older-lady.jpg',
-//     title: 'Doris',
-//     author: 'author',
-//     },
-//     {
-//     img: 'https://previews.123rf.com/images/natakot/natakot1810/natakot181001281/110094284-happy-old-man-showing-thumbs-up-working-out-with-a-dumbbell.jpg',
-//     title: 'Ralph',
-//     author: 'author',
-//    },
-//     {
-//     img: 'https://erinwritesnow.files.wordpress.com/2017/01/cute-happy-older-lady.jpg',
-//     title: 'Doris',
-//     author: 'author',
-//     },
-//    {
-//     img: 'https://erinwritesnow.files.wordpress.com/2017/01/cute-happy-older-lady.jpg',
-//     title: 'Doris',
-//     author: 'author',
-//     },
-//     {
-//     img: 'https://previews.123rf.com/images/natakot/natakot1810/natakot181001281/110094284-happy-old-man-showing-thumbs-up-working-out-with-a-dumbbell.jpg',
-//     title: 'Ralph',
-//     author: 'author',
-//    },
-//  ];
  
-export default function SingleLineGridList({availableResidents, semesterAvailable}) {
+export default function Thumbnails({availableResidents, allResidents, selectResident, semesterAvailable}) {
   const classes = useStyles();
-
-  return (
-    <div className={classes.root}>
-      <GridList className={classes.gridList} cols={2.5}>
-        {availableResidents.map((resident) => (
-          <GridListTile key={resident.photo}>
-            <img src={resident.photo} alt={resident.name} />
-            <GridListTileBar
-              title={resident.name}
-              classes={{
-                root: classes.titleBar,
-                title: classes.title,
-              }}
-              actionIcon={
-                <IconButton aria-label={`star ${resident.name}`}>
-                  {/* <StarBorderIcon className={classes.title} /> */}
-                </IconButton>
-              }
-            />
-          </GridListTile>
-        ))}
-      </GridList>
-    </div>
-  );
+ 
+  const handleClick = (resident) => { 
```

Comment:
So happy that Stephany was able to help with this! Never Forget! 🎆 Holiday fruitcakes for life Steph!
It was as simple as replacing event with resident!

Reply:
Never Forget!